postgres/init.sls

```diff
@@ -59,6 +59,9 @@ postgresql-conf:
         {{ postgres.postgresconf|indent(8) }}
     - show_changes: True
     - append_if_not_found: True
+    {% if not postgres.postgresconf_backup|default(True) -%}
```

Comment:
Habit. There is no real need for it as the default is already provided.  I will remove it at some stage.